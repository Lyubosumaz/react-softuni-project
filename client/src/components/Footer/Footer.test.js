import { shallow } from 'enzyme';
// import Footer from './Footer';

it('should renter Footer Component', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toBeTruthy();
});

it('should checks for html changes', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
});
