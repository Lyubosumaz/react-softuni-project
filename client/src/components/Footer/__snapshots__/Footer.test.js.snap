// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should checks for html changes 1`] = `
<div
  className="main-footer clearfix"
>
  <p
    className="copy-write"
  >
    Â© SoftUni Lyubosumaz Final Project
  </p>
  <ul
    className="footer-nav clearfix"
  >
    <li>
      <button
        className="footer-nav-button"
        onClick={[Function]}
      >
        Home
      </button>
    </li>
    <li>
      <button
        className="footer-nav-button"
        onClick={[Function]}
      >
        About
      </button>
    </li>
  </ul>
</div>
`;
